demos/index.js

```diff
@@ -34,15 +34,25 @@ query(async function () {
             name: 'demo_toolbar',
             items: [
                 { id: 'combo', type: 'radio', text: 'Combo', icon: 'fa fa-star-o', route: 'combo/1' },
+                { type: 'break'},
```

Comment:
break after each toolbar item does not look good (in my opinion). Break make sense to visually separate groups of similar buttons. I would prefer not to have break here.